tests/jballerina-unit-test/src/test/resources/test-src/types/float/float_type_negative.bal

```diff
@@ -0,0 +1,33 @@
+function testInvalidHexaDecimalWithFloatType() {
```

Comment:
Shall check this case too?

```ballerina
type FOO 0xFFFFFFFFFFFFFFFF;
function test() {
    FOO dd = 0xFFFFFFFFFFFFFFFF;
}
```

 it is throwing a Parsing exception.
Also lets write a test case for ReferredTypes

e.g. 
```ballerina
type BOO float;
BOO d = 0xFFFFFFFFFFFFFFFF
```
Second example is giving the expected error.

Reply:
Seems the parsing exception is the same as #32839.

Reply:
Lets merge this fix then, let @rdulmina add the testcase in his PR

Reply:
As per offline discussion, will not be fixing the above issue with this PR.

Reply:
This is partaly fixed in [b0c5a46](https://github.com/ballerina-platform/ballerina-lang/pull/35646/commits/b0c5a46b37e9ced6c47ba77577e7112ac0cc3691) now failing due to  #35695